mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.kt

```diff
@@ -174,40 +229,38 @@ class MockWebServer : ExternalResource(), Closeable {
         .resolve(path)!!
   }
 
-  /**
-   * Sets the number of bytes of the POST body to keep in memory to the given limit.
-   */
-  fun setBodyLimit(maxBodyLength: Long) {
-    this.bodyLimit = maxBodyLength
-  }
-
-  /**
-   * Sets whether ALPN is used on incoming HTTPS connections to negotiate a protocol like HTTP/1.1
-   * or HTTP/2. Call this method to disable negotiation and restrict connections to HTTP/1.1.
-   */
-  fun setProtocolNegotiationEnabled(protocolNegotiationEnabled: Boolean) {
+  @JvmName("-deprecated_bodyLimit")
+  @Deprecated(
+      message = "moved to var",
+      replaceWith = ReplaceWith(
+          expression = "run { this.bodyLimit = bodyLimit }"
+      ),
+      level = DeprecationLevel.WARNING)
+  fun setBodyLimit(bodyLimit: Long) = run { this.bodyLimit = bodyLimit }
```

Comment:
apply instead of run, otherwise you return bodyLimit as the result.  OR should these be returning Unit/Void?

Reply:
These return Unit.